doc/nrf/releases/release-notes-latest.rst

```diff
@@ -247,6 +247,20 @@ The following list summarizes the most important changes inherited from upstream
     * Fixed a discovery response formatting.
     * Updated a few API functions to accept a const pointer when appropriate.
 
+* Bluetooth:
+
+  * Bluetooth host:
+
+    * Fixed a crash when ATT timeout occured on a disconnected ATT channel.
+    * Removed definitions and functions that have been deprecated since the 2.3.0 release or older.
+    * Changed pairing procedure to fail pairing when both side have the same public key.
+    * Fixed an issue where GATT requests might deadlock RX thread.
+    * Fixed not able to clear the fixed passkey previously set.
+    * Fixed callbacks for "security changed" and "pairing failed" not always called.
+    * Changed pairing procedure to fail early if the remote device cannot acheive the required security level.
+    * Fixed incomplete bond overwrite during pairing procedure when peer is not using the IRK stored in the bond.
```

Comment:
```suggestion
    * Fixed a crash where an ATT timeout occurred on a disconnected ATT channel.
    * Removed definitions and functions that were deprecated since the v2.3.0 release.
    * Changed the pairing procedure to fail pairing when both sides have the same public key.
    * Fixed an issue where GATT requests might deadlock RX thread.
    * Fixed an issue where a fixed passkey that was previously set could not be cleared.
    * Fixed an issue where callbacks for "security changed" and "pairing failed" were not always called.
    * Changed the pairing procedure to fail early if the remote device cannot achieve the required security level.
    * Fixed an incomplete bond overwrite during the pairing procedure when the peer is not using the IRK stored in the bond.
```